OpenRA.Game/Server/Server.cs

```diff
@@ -926,7 +926,7 @@ public void SendMessage(string text)
 			DispatchServerOrdersToClients(Order.FromTargetString("Message", text, true));
 
 			if (Type == ServerType.Dedicated)
-				Console.WriteLine($"[{DateTime.Now.ToString(Settings.TimestampFormat)}] {text}");
+				WriteLineWithTimeStamp(ModData.Translation.GetFormattedMessage(text));
```

Comment:
Wouldn't it make more sense to do the translation in `WriteLineWithTimeStamp`?

Reply:
The parameters only align for 2 cases. The third one has different parameters. I extracted the _Game Started_ string, though.